python/tensorflow_macos/Portfile

```diff
@@ -0,0 +1,117 @@
+                    rmd160  812afc151dd87abb657df6d8e66ad8681c91a3bd \
+                    sha256  80807f8b0fc4a98ffa0b7395304fa61239f246a84f1eedbca70a5a1847d4dead \
+                    size    376699291
+
+extract.cmd         ${portutil::autoconf::tar_command}
+extract.pre_args    -xzf
+extract.post_args
+
+if {${os.platform} eq "darwin" && ${os.major} < 20} {
+    pre-fetch {
+        ui_error "TensorFlow with ML Compute acceleration is only available \
+            on macOS 11.0 and later."
+        error {unsupported platform}
+    }
+}
+
+python.versions 38
+
+if {${name} ne ${subport}} {
+    conflicts       py${python.version}-tensorflow \
+                    py${python.version}-tensorflow_addons
+
+    depends_build-append \
+                    port:py${python.version}-pip \
+                    port:py${python.version}-wheel
+
+    depends_run-append \
+                    port:py${python.version}-absl \
+                    port:py${python.version}-astunparse \
+                    port:py${python.version}-flatbuffers \
+                    port:py${python.version}-gast \
+                    port:py${python.version}-grpcio \
+                    port:py${python.version}-h5py \
+                    port:py${python.version}-keras_preprocessing \
+                    port:py${python.version}-numpy \
+                    port:py${python.version}-opt_einsum \
+                    port:py${python.version}-protobuf3 \
+                    port:py${python.version}-tensorflow_estimator \
+                    port:py${python.version}-scipy \
+                    port:py${python.version}-tensorboard \
+                    port:py${python.version}-termcolor \
+                    port:py${python.version}-typing_extensions \
+                    port:py${python.version}-wrapt \
+                    port:py${python.version}-typeguard
+    
+    use_configure   no
+
+    build {}
+
+    destroot.cmd    pip-${python.branch}
+    destroot.args   --ignore-installed \
+                    --no-cache-dir \
+                    --no-dependencies \
+                    --root ${destroot}
+    destroot.post_args
+
+    destroot {
+        foreach whl [glob -type f ${workpath}/${github.project}/x86_64/tensorflow*cp${python.version}*.whl] {
+            system "${destroot.cmd} ${destroot.target} ${destroot.args} ${whl}"
+        }
+    }
+
+    post-destroot {
+        # avoid conflict with py${python.version}-tensorboard
+        delete      ${destroot}${python.prefix}/bin/tensorboard \
+                    ${destroot}${prefix}/bin/tensorboard-${python.branch}
+
+        set github_project_path [glob -nocomplain ${workpath}/${github.author}-${github.project}-*]
+        set docdir ${prefix}/share/doc/${subport}
+        xinstall -d ${destroot}${docdir}
+        xinstall -m 0644 -W ${github_project_path} LICENSE.txt README.md \
+            ${destroot}${docdir}        
+    }
+
+    notes-append \
```

Comment:
These notes are not really necessary as the conflicts declaration above does something equivalent when a conflict is detected.

Reply:
Thanks. Please see #10625.